hudai/resource.py

```diff
@@ -95,6 +95,15 @@ def http_delete(self, path, **request_params):
 
 
     def _list(self, **query_params):
+        query_params['limit'] = 50
+
+        if 'page' in query_params:
+            page = max(query_params['page'], 0)
```

Comment:
passing in page 1 will result in an offset of 50. The max setting here should be 1 and the next line should set offset to `50 * page - 1`